modules/ROOT/pages/create-waf-policy.adoc

```diff
@@ -54,6 +57,10 @@ The *Disable body scanner* option works the same as it does for the request rule
 ** text/html
 ** text/xml
 ** application/json
++
+* *Detect sensitive information* - You can select this option only if the *Disable body scanner* option is not selected. +
```

Comment:
Comment is for section on line 54 which states that responses > 512 KB are rejected when body scanning is enabled. That is not true. Same as comment for request side.